### PR TITLE
[ALLUXIO-1744] Added the Portuguese version of documentation pages: "Architecture.md"; "Metrics-System.md"; "Powered-By-Alluxio.md"

### PR DESCRIPTION
**Pull request by caadecarvalho**
I'm adding the Portuguese version of documentation pages:

- "Architecture.md"
- "Metrics-System.md"
- "Powered-By-Alluxio.md"

I'm adding back "Architecture.md" because I deleted from PR 2731

### CODE REVIEW

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

File: docs/pt/Architecture.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+O modelo do Alluxio utiliza um único `master` e múltiplos `workers`. Em alto nível, Alluxio pode ser 
+divido em três componentes:  o [master](#master); [workers](#worker); e [clients](#client). Os 
+`master` e `workers` juntos formam o Alluxio Server, estes são os componentes que um administrador de
+sistema irá utilizar para gerenciar e manter o sistema. Os `clients` são, genericamente, as aplicações, 
+como as tarefas Spark ou MapReduce, ou usuários utilizando linha de comando (CLI) do Alluxio. Estes 
+usuários irão apenas precisar de uma porção do client do Alluxio.
+
+### Master
+
+Aluxio pode ser disponibilizado em dois modos de `master`, [único master](Running-Alluxio-Locally.html)
```
Comment (aaudiber) [inline docs/pt/Architecture.md]: These links don't work currently - lets either add "../en" to link to the English versions, or translate those pages first.

File: docs/pt/Architecture.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ou [modo de tolerância a falha](Running-Alluxio-Fault-Tolerant-on-EC2.html). O `master` é primeiramente
+responsável por gerenciar a estrutura de metadados global do sistema, como exemplo, a estrutura
+árvore de diretório. Os `clients` podem interagir com o `master` para leitura e modificação do metadado.
+Além disso, todos os `workers` efetuam periodicamente um `heartbeat` para o `master` a fim de manter a 
+participação destes no cluster. O `master` não inicia comunicação com nenhum componente, este apenas 
+interage com outros componentes através de respostas de requisições.
+
+### Worker 
+
+Alluxio `workers` são responsáveis por [gerenciar de recursos locais](Tiered-Storage-on-Alluxio.html)
```
Comment (aaudiber) [inline docs/pt/Architecture.md]: same for this link

File: docs/pt/Architecture.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+alocados para o Alluxio. Estes recursos podem ser memórias locais, discos SSDs ou HDs, que são 
+optados e configurados pelo usuário. Estes `workers` armazenam dados como blocos e respondem requisições 
+de clients para leitura ou escrita de dados através da leitura ou escrita de novos blocos. Entretanto, 
+o `worker` é apenas responsável pelo dado nestes blocos, o mapeamento atual de um arquivo para blocos 
+é armazenado somente no `master`.
+
+### Client
+
+O Alluxio `client` provê um elo de comunicação para interagir com o Alluxio Server. Este provê um 
+[file system API](File-System-API.html) que se comunica com o `master` para tratar das operações de
```
Comment (aaudiber) [inline docs/pt/Architecture.md]: same for this link

File: docs/pt/Architecture.md (hunk 1 above)
Comment (caadecarvalho) [inline docs/pt/Architecture.md]: Thanks, @aaudiber . I'll get this done and I'll let you know.

File: docs/pt/Architecture.md (hunk 2 above)
Comment (caadecarvalho) [inline docs/pt/Architecture.md]: OK.

File: docs/pt/Architecture.md (hunk 3 above)
Comment (caadecarvalho) [inline docs/pt/Architecture.md]: OK.

**Comment by caadecarvalho** [comment]
@aaudiber , I guess that most of the files are here.

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

File: docs/pt/Command-Line-Interface.md [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+poderão ficar com uma baixa capacidade de armazenamento e não efetuar a fixação de demais arquivos.
+
+Como exemplo, o `pin` pode ser utilizado manualmente para garantir performance, se o administrador compreender  
+bem a sua carga de trabalho e rotinas.
+
+{% include Command-Line-Interface/pin.md %}
+
+## report
+O comando `report` marca um arquivo como perdido para o Alluxio `master`. Este comando só deve ser 
+utilizado com arquivos criados utilizando a [Lineage API](Lineage-API.html). Marcando um arquivo como
```
Comment (aaudiber) [inline docs/pt/Command-Line-Interface.md]: broken link - please either prepend `../en` or add `pt/Lineage-API.html`.

**Comment by aaudiber** [comment]
Thanks for translating so many files @caadecarvalho! I went through and found just one broken link - otherwise LGTM

**Comment by caadecarvalho** [comment]
Thanks for the review, @aaudiber. I've pushed the `pt/Lineage-API.md` . Best.

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

File: docs/pt/Command-Line-Interface.md (hunk 4 above)
Comment (caadecarvalho) [inline docs/pt/Command-Line-Interface.md]: OK.

**Comment by aaudiber** [comment]
+1 LGTM

**Pull request merged**